tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/ifelse/TypeGuardTest.java

```diff
@@ -56,8 +56,8 @@ public void testTypeGuardNegative() {
         BAssertUtil.validateError(negativeResult, i++,
                                   "incompatible types: '(float|boolean)' will not be matched to 'string'", 53, 16);
         BAssertUtil.validateError(negativeResult, i++, "unreachable code", 84, 5);
-        BAssertUtil.validateError(negativeResult, i++,
-                                  "incompatible types: '(string|int)' will not be matched to 'float'", 91, 23);
+//        BAssertUtil.validateError(negativeResult, i++,
```

Comment:
Do we have an issue to fix these, https://github.com/ballerina-platform/ballerina-spec/issues/724#issuecomment-771358941

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/29195.